assets/less/hel/hearing-page.less

```diff
@@ -225,7 +112,6 @@
         font-size: 20px;
     }
     .commentlist {
-        background-color: @brand-gray-light;
         padding: 15px;
```

Comment:
Here, I would restore the gray background for the section comments until each section is on a separate page. Without the background there is no visual distinction between section comments and hearing comments that follow them.